src/home-page/components/GridContainer.tsx

```diff
@@ -0,0 +1,18 @@
+import styled from 'styled-components';
```

Comment:
Just a suggestion or food for thought: Right now we only use this nice grid container for `HomePage`, but it could be easily reused in other places of the app if needed. What do you think about moving it to a shared folder? But leaving it here for now (as long as we don't reuse it elsewhere) is perfectly fine, too :)